toolchain/sem_ir/type.h

```diff
@@ -63,6 +64,24 @@ class TypeStore : public Yaml::Printable<TypeStore> {
   // Returns the instruction used to define the specified type.
   auto GetAsInst(TypeId type_id) const -> Inst;
 
+  // Converts an ArrayRef of `InstId`s to a range of `TypeInstId`s via
+  // GetAsTypeInstId().
+  auto GetBlockAsTypeInstIds(llvm::ArrayRef<InstId> array
+                             [[clang::lifetimebound]]) const {
```

Comment:
I think we usually use an explicit trailing `-> auto` to distinguish intentional return type deduction from a forgotten return type.

Reply:
Ah okay, will do